SLI_README.md

```diff
@@ -19,23 +19,11 @@ To measure SLI, please follow all the steps below:
 1. Install [GCP Census](https://github.com/ocadotechnology/gcp-census) application that periodical collects metadata about BigQuery tables. 
 1. Configure [Cloud Datastore export](./SETUP.md#cloud-datastore-export), 
 which periodically exports backup metadata and stores it in BigQuery,
-1. Install [Terraform](https://www.terraform.io/) if it's not available,
-1. Create [Terraform backend](https://www.terraform.io/docs/backends/) GCS bucket, which will be used to store TF state:
-      ```bash
-      export TERRAFORM_STATE_BUCKET_ID="<your-bucket-id-to-store-terraform-state>"
-      gsutil mb -p ${BBQ_PROJECT_ID} gs://${TERRAFORM_STATE_BUCKET_ID}/
-      ```
-1. Export BBQ, BBQ restoration project and GCP Census project ids: 
-      ```bash
-      export TF_VAR_bbq_project=${BBQ_PROJECT_ID}
-      export TF_VAR_bbq_restoration_project=${BBQ_RESTORATION_PROJECT_ID}
-      export TF_VAR_gcp_census_project=${GCP_CENSUS_PROJECT_ID}
-      ```
-1. Create a bucket to store remotely infrastructure state and then create all views/tables using [Terraform](https://www.terraform.io/) by running the following commands:
-      ```bash
-      terraform init -backend-config="bucket=${TERRAFORM_STATE_BUCKET_ID}"
-      terraform apply
-      ```
+1. Install Terraform as in [terraform setup file](TERRAFORM_SETUP.md).
```

Comment:
It should be "Install and initialise Terraform"

Reply:
👍